enterprise/models.py

```diff
@@ -379,6 +379,12 @@ class Meta:
         default=False,
     )
 
+    enable_learner_credit_modal = models.BooleanField(
```

Comment:
As discussed let's change the name of this field to make it more clear what it's referring to, maybe `enable_learner_credit_summary` or something to more closely match the frontend